agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AppInsightsCertificate.java

```diff
@@ -0,0 +1,121 @@
+          + System.lineSeparator()
+          + "MRUwEwYDVQQKEwxEaWdpQ2VydCBJbmMxGTAXBgNVBAsTEHd3dy5kaWdpY2VydC5j"
+          + System.lineSeparator()
+          + "b20xIDAeBgNVBAMTF0RpZ2lDZXJ0IEdsb2JhbCBSb290IEcyMIIBIjANBgkqhkiG"
+          + System.lineSeparator()
+          + "9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuzfNNNx7a8myaJCtSnX/RrohCgiN9RlUyfuI"
+          + System.lineSeparator()
+          + "2/Ou8jqJkTx65qsGGmvPrC3oXgkkRLpimn7Wo6h+4FR1IAWsULecYxpsMNzaHxmx"
+          + System.lineSeparator()
+          + "1x7e/dfgy5SDN67sH0NO3Xss0r0upS/kqbitOtSZpLYl6ZtrAGCSYP9PIUkY92eQ"
+          + System.lineSeparator()
+          + "q2EGnI/yuum06ZIya7XzV+hdG82MHauVBJVJ8zUtluNJbd134/tJS7SsVQepj5Wz"
+          + System.lineSeparator()
+          + "tCO7TG1F8PapspUwtP1MVYwnSlcUfIKdzXOS0xZKBgyMUNGPHgm+F6HmIcr9g+UQ"
+          + System.lineSeparator()
+          + "vIOlCsRnKPZzFBQ9RnbDhxSJITRNrw9FDKZJobq7nMWxM4MphQIDAQABo0IwQDAP"
+          + System.lineSeparator()
+          + "BgNVHRMBAf8EBTADAQH/MA4GA1UdDwEB/wQEAwIBhjAdBgNVHQ4EFgQUTiJUIBiV"
+          + System.lineSeparator()
+          + "5uNu5g/6+rkS7QYXjzkwDQYJKoZIhvcNAQELBQADggEBAGBnKJRvDkhj6zHd6mcY"
+          + System.lineSeparator()
+          + "1Yl9PMWLSn/pvtsrF9+wX3N3KjITOYFnQoQj8kVnNeyIv/iPsGEMNKSuIEyExtv4"
+          + System.lineSeparator()
+          + "NeF22d+mQrvHRAiGfzZ0JFrabA0UWTW98kndth/Jsw1HKj2ZL7tcu7XUIOGZX1NG"
+          + System.lineSeparator()
+          + "Fdtom/DzMNU+MeKNhJ7jitralj41E6Vf8PlwUHBHQRFXGU7Aj64GxJUTFy8bJZ91"
+          + System.lineSeparator()
+          + "8rGOmaFvE7FBcf6IKshPECBV1/MUReXgRPTqh5Uykw7+U0b6LJ3/iyK5S9kJRaTe"
+          + System.lineSeparator()
+          + "pLiaWN0bfVKfjllDiIGknibVb63dDcY3fe0Dkhvld1927jyNxF1WW6LZZm6zNTfl"
+          + System.lineSeparator()
+          + "MrY="
+          + System.lineSeparator()
+          + "-----END CERTIFICATE-----"
+          + System.lineSeparator();
+
+  AppInsightsCertificate() {}
+
+  boolean isInJavaKeystore() {
+    String loadedCertificates = loadCertificates();
+    return loadedCertificates.contains(APP_INSIGHT_ROOT_CERTIFICATE);
+  }
+
+  @SuppressFBWarnings(
+      value = "SECCI", // Command Injection
+      justification = "No user data is used to construct the command below")
+  private static String loadCertificates() {
+    String keyStoreLocation = System.getProperty("java.home") + "/lib/security/cacerts";
+    return executeWithoutException(
+        new ProcessBuilder("keytool", "-list", "-rfc", "-keystore", keyStoreLocation));
+  }
+
+  private static String executeWithoutException(ProcessBuilder processBuilder) {
+    try {
+      return execute(processBuilder);
+    } catch (RuntimeException e) {
+      String header = "Error when executing command " + processBuilder.command() + ": ";
+      if (e.getSuppressed().length == 1) {
+        return header + e.getMessage() + " (Suppressed: " + e.getSuppressed()[0] + ")";
+      }
+      return header + e.getMessage();
```

Comment:
it doesn't look like this return value is displayed anywhere, what about?

```suggestion
      logger.error(e.getMessage(), e);
```